### PR TITLE
Add `totalBillableCharacters` to `CountTokensResponse`

### PR DESCRIPTION
**Pull request by rlazo**
Also, make the fields optional.

### CODE REVIEW

File: generativeai/src/main/java/com/google/ai/client/generativeai/type/CountTokensResponse.kt [hunk 1]
```diff
@@ -21,6 +21,6 @@ package com.google.ai.client.generativeai.type
  *
  * @property totalTokens A count of the number of tokens in the input
  */
-class CountTokensResponse(val totalTokens: Int) {
+class CountTokensResponse(val totalTokens: Int?) {
```
Comment (daymxn) [inline generativeai/src/main/java/com/google/ai/client/generativeai/type/CountTokensResponse.kt:24]: This seems counter intuitive. Is `totalBillableCharacters` not public yet?

Comment (rlazo) [reply generativeai/src/main/java/com/google/ai/client/generativeai/type/CountTokensResponse.kt:24]: Not yet

Comment (daymxn) [reply generativeai/src/main/java/com/google/ai/client/generativeai/type/CountTokensResponse.kt:24]: If it's not public yet, then isn't totalTokens not nullable yet from the publics perspective? Also, could `totalBillableCharacters` *and* `totalTokens` both have values at the same time- or are they xor?

Comment (rlazo) [reply generativeai/src/main/java/com/google/ai/client/generativeai/type/CountTokensResponse.kt]: Agree, fixed.

**Review by daymxn — APPROVED** [review]

**Pull request merged**